libs/cypress/src/lib/eip1193-bridge.ts

```diff
@@ -113,6 +113,12 @@ const getProvider = () =>
     Cypress.env('ethereumChainId')
   );
 
+const getRandomKey = () => {
+  const createRandomWallet = Wallet.createRandom();
+  const randomPrivateKey = createRandomWallet.mnemonic.phrase;
+  return Wallet.fromMnemonic(randomPrivateKey, getAccount(0)).privateKey;
```

Comment:
chore (blocking): don't use a random key, rather use the same mnemonic and generate the second key from that mnemonic. 

In the `createBridge` function below I would allow it to take an integer as a parameter, which should be passed to the `getAccount` function instead of 0.

Then you can take the account number as a parameter in the `mockWeb3Provider` command in  `mock-web3-provider.ts` and for the tests where you need to be unapproved you set it so that for example.

key 0, used for trading and has everything enabled. This is the default unless specified
key 1, used to test forms for approval ect.

and then you can continue to add keys infinitely all from the same mnemonic.

It would be good to document what key is used for what in a README.md file somewhere :+1: